src/main/java/com/conveyal/datatools/manager/models/FeedSourceSummary.java

```diff
@@ -42,6 +42,9 @@ public class FeedSourceSummary {
     public boolean deployable;
     public boolean isPublic;
 
+    /** An optional display filename for the feed in the bundle, e.g. "agency_transit.zip" */
+    public String filename;
+
```

Comment:
Do we need this in the summary? Isn't it fine if the data only loads on requesting the entire feed source

Reply:
I was thinking you would want to see all the feed filenames at a glance without having to click into each of them when you are writing a build config